index.rst

```diff
@@ -253,9 +253,9 @@ and the supported values associated with them:
      - Set to ``true`` ensure file ends with a newline when saving and ``false``
        to ensure it doesn't.
    * - ``root``
-     - Must be specified in the preamble. Set to ``true`` to stop the
-       ``.editorconfig`` file search on the current file. The value is case
-       insensitive.
+     - Must be specified in the preamble.  Case-insensitive.
```

Comment:
I think it's still better to make it clear that the value is case-insensitive. Now it's slightly ambiguous, since the first sentence seems to refer to the key-value pair.